packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -38,11 +54,10 @@ const getIPAddress = async (url: string): Promise<string> => {
   });
 };
 
-export const createHandler = async ({ input, ctx }: CreateOptions) => {
-  const { slug, name, orgOwnerEmail, seats, pricePerSeat, isPlatform } = input;
+const checkLoginStatus = async (userId: number) => {
```

Comment:
Better name would be `getUserOrThrow`.

The function doesn't seem to check anything related to login. It is just getting a user.